### ISSUE TITLE
magento: command not found

### ISSUE THREAD

**Issue by ghost** [davidalger/capistrano-magento2#50]
Hello,

I'm having issue during static content deploy. See below:

DEBUG [023dc095] Command: cd /var/www/html/project/releases/20161123062111 && set -o pipefail; magento setup:static-content:deploy en_US -t Smartwave/porto -t Smartwave/porto_rtl -t Kartcent/default -t Magento/backend | stdbuf -o0 tr -d .

 DEBUG [023dc095] 	bash: magento: command not found

**Comment by erikhansen** [comment]
@davidalger I just experienced this issue for the first time. 0.5.7 has been working fine on a Digital Ocean server I've been deploying to, but it doesn't work when deploying to a number of the production environments I manage. Not sure why it _has_ been working, as the command that is being run is:

```
set -o pipefail; magento setup:static-content:deploy en_US | stdbuf -o0 tr -d .
```

It looks like the error may be somewhat related to this SSHKit issue: https://github.com/capistrano/sshkit/issues/48

Obviously it would be ideal to get the nifty prefix functionality to work. However if that proves infeasible, you may have to revert to a workaround like this:

![image](https://cloud.githubusercontent.com/assets/129031/20686420/3c09fbc0-b57e-11e6-8260-58b0029fd39c.png)

**Comment by davidalger** [comment]
@chiboshi2 Thanks for reporting. I've just fixed this in 0.5.8. The bug was due to some odd behavior in how SSHKit treats commands once prefixes are applied and slipped into this update without being noticed due to a shortcut which exists in our test/stage deployment environments.

**Comment by ghost** [comment]
Thanks for the fix @davidalger.
